### PR TITLE
add branch

### PR DESCRIPTION
**Pull request by grasipacheco**

### CODE REVIEW

**Review by kkenjeres — APPROVED** [review]
Good job!

File: GithubAndMMarkdow/notes.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+git switch - goes back to the last used branch
+git switch -c (name of the branch) - creates a new branch
```
Comment (F-Kirchhoff) [inline GithubAndMMarkdow/notes.md]:
```suggestion
git switch -c (name of the branch) - creates a new branch and switches onto it 
```

File: GithubAndMMarkdow/notes.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+steps to send information to the remote repository
+git add
```
Comment (F-Kirchhoff) [inline GithubAndMMarkdow/notes.md]:
```suggestion
git add <file or directory> (e.g. `.`)
```

**Review by F-Kirchhoff — APPROVED** [review]
looks great. I added two suggestions, please feel free to use them or discard them.

**Review by kabaskill — APPROVED** [review]
looks cool!

**Pull request merged**